packages/liveblocks-react/package.json

```diff
@@ -34,7 +34,7 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "@liveblocks/client": "0.15.10",
-    "react": "^16.14.0 || ^17"
+    "react": ">=16.14.0 || ^17 || ^18"
```

Comment:
Yay for supporting React 18!

Reply:
Btw, I think using `>=16.14.0` makes the remainder of the line moot, because all versions higher than 16 also match when using `>=`, including 17, 18, and even (in the future) 19. Is there a reason you changed to `>=` here that I'm missing?

Reply:
It was my initial change and I forgot to revert it. IT's fixed now!